Can't install v0.3.1

Why need git binary?
the error message is as follows:

npm -g install jimp
npm ERR! code ENOGIT
npm ERR! Error while executing:
npm ERR! undefined ls-remote -h -t ssh://git@github.com/makr28/jpgjs.git
npm ERR!
npm ERR! undefined
npm ERR! No git binary found in $PATH
npm ERR!
npm ERR! Failed using git.
npm ERR! Please check if you have git installed and in your PATH.

--- a/package.json
+++ b/package.json
@@ -10,7 +10,8 @@
     "browser",
     "dist",
     "es",
-    "jimp.d.ts"
+    "jimp.d.ts",
+    "fonts"
   ],
   "repository": {
     "type": "git",

--- a/src/constants.js
+++ b/src/constants.js
@@ -44,7 +44,7 @@ function getJimpDir() {
     const reMoz = /.*@([^?]+\/).*/;
 
     if (process.env.BABEL_ENV === 'test' && process.env.ENV === 'browser') {
-        return 'http://localhost:9877/base/test/';
+        return 'http://localhost:9876/base/test/';
     }
 
     if (reWebKit.test(callLine)) {

--- a/test/async.test.js
+++ b/test/async.test.js
@@ -10,7 +10,7 @@ describe('Async functions', () => {
 
         // process.env is undefined in the browser tests. If BABEL_ENV
         // isn't found don't run this test in the browser
-        if (process.env.BABEL_ENV === undefined) {
+        if (process.env.ENV === 'browser') {
             return done();
         }
 
@@ -27,7 +27,7 @@ describe('Async functions', () => {
     });
 
     it('getBuffer returns promise', done => {
-        if (process.env.BABEL_ENV === undefined) {
+        if (process.env.ENV === 'browser') {
             return done();
         }
 
@@ -41,7 +41,7 @@ describe('Async functions', () => {
     });
 
     it('getBase64 returns promise', done => {
-        if (process.env.BABEL_ENV === undefined) {
+        if (process.env.ENV === 'browser') {
             return done();
         }
 

